src/stdatamodels/model_base.py

```diff
@@ -263,6 +286,17 @@ def __init__(self, init=None, schema=None, memmap=False,
                 if 'datatype' in subschema:
                     setattr(self, attr, value)
 
+    @property
+    def schema_url(self):
+        """
+        The schema URI to validate the model against.  If
+        None, no validation will occur.
+        Returns
+        -------
+        str or None
+        """
+        return None
```

Comment:
Will this property have to be overwritten by any subclasses that have a `schema_url`?

Reply:
Yes, though this is a valid way to overwrite it:

```
class FooModel(DataModel):
    schema_url = "http://stsci.edu/schemas/foo"
```

so this should be compatible with the existing jwst model classes.  The only reason why I went with a property over a class attribute was so I could add a docstring.

Reply:
I'll switch this back to an attribute